tests/page/page-network-response.spec.ts

```diff
@@ -270,14 +268,14 @@ it('should behave the same way for headers and allHeaders', async ({ page, serve
   expect(allHeaders['name-b']).toEqual('v4');
 });
 
-it('should provide a Response with a file URL', async ({ page, asset, isAndroid, isElectron, isWindows, browserName, browserMajorVersion, mode }) => {
+it('should provide a Response with a file URL', async ({ page, asset, isAndroid, isWindows, browserName, browserMajorVersion, mode }) => {
   it.skip(isAndroid, 'No files on Android');
   it.skip(browserName === 'firefox', 'Firefox does return null for file:// URLs');
   it.skip(mode.startsWith('service'));
 
   const fileurl = url.pathToFileURL(asset('frames/two-frames.html')).href;
   const response = await page.goto(fileurl);
-  if (isElectron || (browserName === 'chromium' && browserMajorVersion >= 99) || (browserName === 'webkit' && isWindows))
+  if ((browserName === 'chromium' && browserMajorVersion >= 99) || (browserName === 'webkit' && isWindows))
```

Comment:
This change is a bit surprising. I'd think that new electron behaves like new chromium and returns `200`.